skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/utils/MentionUtils.scala

```diff
@@ -44,7 +44,7 @@ object MentionUtils {
 
   def getMentionsWithoutLocations(texts: Seq[String], file: File, reader: OdinEngine): Seq[Mention] = {
     // this is for science parse
-    val mentions = texts.flatMap(t => reader.extractFromText(t, filename = Some(file.getName)))
+    val mentions = texts.flatMap(t => reader.extractFromText(t, filename = Some(file.getName))._2)
```

Comment:
I wish that `extractFromText` returned something other than a tuple.  In other projects we've called it an `AnnotatedDocument`.  Then we don't have to do the by the number _._2 thing.

Reply:
Good point. I changed the tuple for a case class